src/EnergyPlus/SizingManager.cc

```diff
@@ -1776,13 +1778,21 @@ namespace EnergyPlus::SizingManager {
                                                                  TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName,
                                                                  VpzMinHtgByZone(termUnitSizingIndex),
                                                                  4); // Vpz-min
+                        Real64 VozHtg = 0.0;
                         if (TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneADEffHeating != 0.0) {
+                            VozHtg = VbzByZone(termUnitSizingIndex) / TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneADEffHeating;
                             OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchS62zhdVozhtg,
                                                                      TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneName,
-                                                                     VbzByZone(termUnitSizingIndex) /
-                                                                         TermUnitFinalZoneSizing(termUnitSizingIndex).ZoneADEffHeating,
+                                                                     VozHtg,
                                                                      4); // Voz-htg
                         }
+                        // Outdoor Air Details Report - Design Zone Outdoor Airflow - Voz
+                        Real64 VozMax = std::max(VozHtg, VozClg);  // take larger of the heating and cooling Voz values
```

Comment:
During sizing, collect design outdoor air flow for the Mechanical Ventilation Parameters by Zone subtable.